server/handler.go

```diff
@@ -91,3 +93,50 @@ func (s Server) getAuthProviderClient(org, commonName string) (auth.Interface, e
 
 	return nil, errors.Errorf("Client is using unknown organization %s", org)
 }
+
+func (s Server) Authzhandler(w http.ResponseWriter, req *http.Request) {
+	if req.TLS == nil || len(req.TLS.PeerCertificates) == 0 {
+		write(w, nil, WithCode(errors.New("Missing client certificate"), http.StatusBadRequest))
+		return
+	}
+	crt := req.TLS.PeerCertificates[0]
+	if len(crt.Subject.Organization) == 0 {
+		write(w, nil, WithCode(errors.New("Client certificate is missing organization"), http.StatusBadRequest))
+		return
+	}
+	org := crt.Subject.Organization[0]
+	glog.Infof("Received subject access review request for %s/%s", org, crt.Subject.CommonName)
+
+	data := authzv1.SubjectAccessReview{}
+	err := json.NewDecoder(req.Body).Decode(&data)
+	if err != nil {
+		write(w, nil, WithCode(errors.Wrap(err, "Failed to parse request"), http.StatusBadRequest))
+		return
+	}
+
```

Comment:
add `defer req.Body.Close()`
Look like it's missed in token review handler as well. can you add it?